src/MiNET/MiNET/Net/RakNet/RakOfflineHandler.cs

```diff
@@ -84,7 +84,7 @@ internal void HandleOfflineRakMessage(ReadOnlyMemory<byte> receiveBytes, IPEndPo
 			// Shortcut to reply fast, and no parsing
 			if (messageType == DefaultMessageIdTypes.ID_OPEN_CONNECTION_REQUEST_1)
 			{
-				if (!_greyListManager.AcceptConnection(senderEndpoint.Address))
```

Comment:
I'd still like to see this be put back as it was.

Reply:
Still waiting for an explanation why this shouldn't be, since the old implementation doesn't allow whitelist clients by Endpoint and not Ip but at least for me, this feature is critically important. It doesn't affect performance or anything else in any way.